### PR TITLE
Use latest CC release in preparation for beta1 release

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: app/views/my/_index_partials/_default_group.html.erb [hunk 1]
```diff
@@ -12,9 +12,9 @@
   </thead>
   <% end %>
   <tbody>
-  <% docs.each_with_index do |document,counter| %>
+  <% docs.each_with_index do |document, counter| %>
     <% case document.hydra_model %>
-    <% when "Collection" %>
+    <% when ::Collection %>
```
Comment (jcoyne) [inline app/views/my/_index_partials/_default_group.html.erb]: how about `if document.collection?`

**Comment by jcoyne** [comment]
👮 🔴 (I've been waiting for the day your PR gets stopped by Rubocop 😉 )

**Comment by mjgiarlo** [comment]
Damn.

**Comment by mjgiarlo** [comment]
@jcoyne silenced the cop. Build is still gonna be broken, but no worries if you're gonna cut beta7 and take this up tomorrow. Gotta :runner:

**Comment by mjgiarlo** [comment]
Subsequent to this PR, in order to use nested works in the Sufia UI, what do we need? A quick glance reveals:

- [ ] Include the cc/base/form_in_works partial in our cc/base/form override
- [ ] Merge changes made to CC/base/show_actions partial in our override
- [ ] Merge changes made to CC/base/show template in our override
- [ ] :question:

**Pull request merged**